 Project1/index.html

```diff
@@ -0,0 +1,99 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>
+        Yes, the Octopus Is Smart as Heck. But Why?
+    </title>
+    <link rel="stylesheet" href="style.css">
+</head>
+
+<body>
+
+    <figure>
+        <img src="images/Illustration of Octopus.jpg" alt="desciption of photo" width="800" height="420">
```

Comment:
You should always rename your filenames to **not** have spaces, since the spaces can break your images in some browsers. Note that you have update the `src` here and _also_ the actual filename in your `images` directory.

```suggestion
        <img src="images/octopus.jpg" alt="desciption of photo" width="800" height="420">
```